### PR TITLE
Update migrations.rst

### PR DESCRIPTION
**Pull request by jeffblack360**
Added a note about the cakephp/migrations 1.5.2 and Camelizing the migration file name.
I also updated the sample migrations code at the top of the document.

### CODE REVIEW

File: en/migrations.rst [hunk 1]
```diff
@@ -129,21 +140,25 @@ The following ``Bake`` command would create a migration to add a ``products`` ta
...
+        $ bin/cake bake migration create_products name:string description:text created modified
+    
+        Welcome to CakePHP v3.0.13 Console
+        ---------------------------------------------------------------
+        App : src
+        Path: /home/user/Work/php/cakeblog/src/
+        ---------------------------------------------------------------
+        
+        Creating file /home/user/Work/php/cakeblog/config/Migrations/20160121164955_create_products.php
+        Wrote `/home/user/Work/php/cakeblog/config/Migrations/20160121164955_create_products.php`
```
Comment (HavokInspiration) [inline en/migrations.rst]: Shouldn't the filenames be camelized in the output example ?

Comment (jeffblack360) [inline en/migrations.rst]: :+1: Ahh you're right. Before we began this conversation I wanted to demonstrate that you could still see the underscore form, but with the version added note I suppose that is moot.
I'll change it!

File: en/migrations.rst [hunk 2]
```diff
@@ -129,21 +140,25 @@ The following ``Bake`` command would create a migration to add a ``products`` ta
...
-            
-                Welcome to CakePHP v3.0.13 Console
-                ---------------------------------------------------------------
-                App : src
-                Path: /home/user/Work/php/cakeblog/src/
-                ---------------------------------------------------------------
-                
-                Creating file /home/user/Work/php/cakeblog/config/Migrations/20160121164955_create_products.php
-                Wrote `/home/user/Work/php/cakeblog/config/Migrations/20160121164955_create_products.php`
+    Camelizing the file name of the initial migration was introduced in v1.5.2 of the
```
Comment (HavokInspiration) [inline en/migrations.rst]: I'd replace "initial migration" with "migration file".

File: en/migrations.rst [hunk 3]
```diff
@@ -129,21 +140,25 @@ The following ``Bake`` command would create a migration to add a ``products`` ta
...
+    `migrations plugin <https://github.com/cakephp/migrations/>`_. This version of
+    the plugin is only available with a release of CakePHP >= to 3.1.
```
Comment (HavokInspiration) [inline en/migrations.rst]: And here, add a note that prior to this version of the plugin, the name would be in the underscored form.

Comment (jeffblack360) [inline en/migrations.rst]: Will do.

File: en/migrations.rst (hunk 2 above)
Comment (jeffblack360) [inline en/migrations.rst]: Will do. I added "initial" to it because of your commit message on the plugin.

**Comment by jeffblack360** [comment]
@HavokInspiration Looks good! :+1:

**Comment by HavokInspiration** [comment]
Good work @jeffblack360 :+1:

**Pull request merged**